app/controllers/turbo/android/path_configurations_controller.rb

```diff
@@ -0,0 +1,74 @@
+module Turbo
+  module Android
+    class PathConfigurationsController < ApplicationController
+      def show
+        render json: {
+          settings: {
+            # Tabs are serialized to a string. RE: https://github.com/hotwired/turbo-android/issues/209
+            tabs: [
+              {
+                title: "Home",
+                path: root_path,
+                image_url: svg_icon_path("home")
+              },
+              {
+                title: "Posts",
+                path: posts_path,
+                image_url: svg_icon_path("compose")
+              }
+            ].to_json
+          },
+          rules: [
+            {
+              patterns: [".*"],
+              properties: {
+                context: "default",
+                uri: "turbo://fragment/web",
+                fallback_uri: "turbo://fragment/web",
+                pull_to_refresh_enabled: true
+              }
+            },
+            {
+              patterns: ["^$", "^/$"],
+              properties: {
+                uri: "turbo://fragment/web/home",
+                presentation: "replace_root"
+              }
+            },
+            {
+              patterns: ["/new$", "/edit$"],
+              properties: {
+                context: "modal",
+                uri: "turbo://fragment/web/modal/sheet",
+                pull_to_refresh_enabled: false
+              }
+            },
+            {
+              patterns: ["/users/sign_in"],
+              properties: {
+                uri: "turbo://fragment/users/sign_in",
+                context: "modal"
+              }
+            },
+            # TODO: Needs a /api/v1/users_controller.rb for registration to work
+            # {
+            #   patterns: ["/users/sign_up"],
+            #   properties: {
+            #     uri: "turbo://fragment/users/sign_up",
+            #     context: "modal"
+            #   }
+            # }
```

Comment:
I vote we remove these comments and add a new issue to show that it's on the roadmap.

Reply:
FWIW, I can add the registration flow later next week.

Reply:
Sounds good. I'll add an issue and remove the commented out code.

Reply:
#15 created, and commented out code has been removed.

Reply:
Awesome, thanks!